release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1061,5 +1068,165 @@ module openconfig-evpn {
 
       uses evpn-vxlan-parameters-config;
     }
+
+    container endpoint-peers {
+      description "Overlay endpoint peers oper data";
+      config false;
+
+      list endpoint-peer {
+        key "peer-address";
+          description "Overlay endpoint peers oper data";
+
+        leaf peer-address {
+        type leafref{
+          path '../state/peer-address';
+        }
+        description "overlay endpoint peer ip address";
+        }
+        container state {
+         config false;
+         description
+           "State parameters relating to the overlay tunnel endpoints
+           of the network instance";
+           uses evpn-endpoint-peer-state;
+        }
+      }
+    }
+
+    container endpoint-vnis {
+      description "Overlay endpoint peers oper data";
+      config false;
+
+      list endpoint-vni {
+        key "vni";
+        description "Overlay endpoint evis oper data";
+
+        leaf vni {
+          type leafref{
+            path '../state/vni';
+            }
+            description "VNI Identifier";
+        }
+        container state {
+          config false;
+           description
+             "State parameters relating to the overlay tunnel endpoints
+             of the network instance";
+             uses evpn-endpoint-vni-state;
+        }
+      }
+    }
+  }
+
+  grouping evpn-endpoint-peer-state {
+    description
+    "Grouping for VXLAN EVPN Overlay endpoint oper data";
+    leaf peer-address {
+      type oc-inet:ip-address;
+      description "VXLAN EVPN Overlay endpoint peer ip address";
```

Comment:
For all of the descriptions here, please can we convert them to full sentences, with some additional description if it is needed - i.e., things like "Peer Router mac-address" could do with having clearer description of what peer router that is.

The general rule of thumb is that the descriptions should be sufficient to write some of test against the leaf to ensure that it implements what is expected. The documentation for the models is also auto-generated here, so the more descriptive we can be, the better.